src/backend/llm/model.py

```diff
@@ -23,7 +23,7 @@ def summarize(text):
 
     try:
         response = Groq(api_key=GROQ_API_KEY).chat.completions.create(
-            messages=message, model='llama3-70b-8192'
+            messages=message, model='llama-3.1-8b-instant'
```

Comment:
This model is newer and has a higher token limit. However, it is just the 8b Model which is not as good as a 70b or the newest 405b model. However, I have no access to those models yet.